doc/installation/environment_vars.rst

```diff
@@ -72,3 +72,23 @@ Environment variables for software developement
 +-------------------------------+---------------------------------------------------------------------------------------+
 | PYTHONPATH                    | List of directories searched for python modules/packages                              |
 +-------------------------------+---------------------------------------------------------------------------------------+
+
+Environment variables affecting OpenMP
+--------------------------------------
+
+Certain environment variables will affect the behavior of OpenMP.
+In particular they allow you to set the number of OpenMP threads that will be spawned.
+
+.. warning:: This is of particular importance when using MPI, as each MPI rank will potentially spawn
+             an equal number of OpenMP threads. If you intend to use as MPI ranks as cores are available
+             you should limit the number of OpenMP threads to one!
+
++-------------------------------+---------------------------------------------------------------------------------------+
+| Environment Variable          | Brief Description                                                                     |
++===============================+=======================================================================================+
+| OMP_NUM_THREADS               | Specifies globally the number of OpenMP threads to use                                |
++-------------------------------+---------------------------------------------------------------------------------------+
+| MKL_NUM_THREADS               | Specifies the number of OpenMP threads to use only within the Intel MKL library       |
++-------------------------------+---------------------------------------------------------------------------------------+
+| OPENBLAS_NUM_THREADS          | Specifies the number of OpenMP threads to use within the OpenBlas library             |
++-------------------------------+---------------------------------------------------------------------------------------+
```

Comment:
Should we add something explicitly saying that if you don't intend to use OpenMP you should set OMP_NUM_THREADS=1 in your code (I don't remember if this is stated somewhere else)?